### PR TITLE
Add deploy keys

### PR DESCRIPTION
**Pull request by snovikov**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -271,5 +281,5 @@ variable "webhooks" {
     })
   }))
   default     = []
-  description = "List of webhook configurations"
+  description = "List of webhook configurations."
```
Comment (danvaida) [inline variables.tf:284]: I am more in favor of avoiding the dot (here, signifying the end of a sentence) in such use cases, for the same reason as leaving it out in the commit message. When there are more sentences, I use it. But when there's only one, this extra character serves no purpose, IMO.

File: examples/init_repo_with_deploy_keys/main.tf [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  license_template   = "mit"
+
+  vulnerability_alerts = true
+
+  default_branch_protection_enabled = false
+
+  deploy_keys = [
+    {
+      title     = "read-only"
+      key       = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAAAgQCnxrhTA8hy5xje/++8BfvbhuKYCWD9FZqNs99BXkur/2GOOw8JX3ClRPnxmFSP3wUekarv0FgIiPrUhE4g4ea1CCoqC3HhZmfoG+VsvjhJnn+LjhcVIh6MvlQ42ZixJCdH3vmLm13BVZMzFfhW1v6pYfm3hkjU7UQvMHUGiLEpvw=="
```
Comment (danvaida) [inline examples/init_repo_with_deploy_keys/main.tf:37]: In a future iteration, it'd be great to enforce a key-length... although I'm not sure we can or should do this, here. Probably in some other GitHub settings.

**Review by danvaida — APPROVED** [review]
👍

File: variables.tf (hunk 1 above)
Comment (snovikov) [reply variables.tf:284]: I think it is OK to leave it here

File: examples/init_repo_with_deploy_keys/main.tf (hunk 2 above)
Comment (snovikov) [reply examples/init_repo_with_deploy_keys/main.tf:37]: I think you cannot enforce it on GitHub side. But we can add some validation on top for this input paramters.

**Pull request merged**